Sources/Workspace/Destination.swift

```diff
@@ -57,16 +58,16 @@ public struct Destination: Encodable {
     private static func hostBinDir(
         originalWorkingDirectory: AbsolutePath? = localFileSystem.currentWorkingDirectory
     ) -> AbsolutePath {
+        if let toolchainDir = ProcessInfo.processInfo.environment["SWIFT_TOOLCHAIN_PATH"] {
```

Comment:
Looks like there is already a `SWIFTPM_CUSTOM_BINDIR` variable which serves this purpose but it's complied for Debug. We can just get rid of the `#if DEBUG` there.

Reply:
Ah, I missed that, I've enabled that instead.